api/definitions/comment_rejection_object.md

```diff
@@ -0,0 +1,50 @@
+# Comment Rejection Object
+
+*Note: Info here isn't 100% complete, since we don't have sample information for what most comment rejection objects actually look like.*
+
+A comment rejection object looks like this:
+
+```
+{
+    rejected: /* Rejection code - see below */
+    status: /* (optional) Status object with rejection details - see below */
+    appealId: /* (if IP muted) Case number to contact ST with to appeal block */
+}
+```
+
+The `rejected` property will be one of the following:
+
+- `"isEmpty"`: the comment is empty
+- `"isFlood"`: sending too many comments in a short period of time
```

Comment:
I'm assuming this applies to the rate limits on both New Scratcher and Scratcher accounts?

Reply:
I assume so too. We don't have experimental data for the vast majority of this (only have one case of `"isMuted"`), or reference info to refer to elsewhere, except the error messages associated with these error codes: https://github.com/LLK/scratch-www/blob/develop/src/l10n.json#L325-L335

The `"isFlood"` message reads this:

> Woah, seems like you're commenting really quickly. Please wait longer between posts.